pyxrf/view/fit.enaml

```diff
@@ -382,11 +385,13 @@ enamldef ElementEdit(Window):
                             cb_strat3, cb_strat4, cb_strat5, cb_strat6, spacer)
                         ]
                         FloatField: ff_value:
-                            value := float(parameter_model.param_dict[loop_item]['value'])
```

Comment:
Yes. Some values are mistakenly created as int, so I corrected that part. Everything is working now. We do not need to put float here. 
I also find if we put float here, the dict value will not change if value changes from the GUI.